lib/config.js

```diff
@@ -1,6 +1,22 @@
 const _ = require('lodash')
 
 module.exports = {
+  compareSemVer(a, b) {
```

Comment:
I think you can use [semver](https://www.npmjs.com/package/semver) for this. It's already installed.

Reply:
Thanks! Fixed.